sonar-project.properties

```diff
@@ -13,7 +13,7 @@ sonar.links.scm=https://github.com/RGPosadas/Mull
 sonar.host.url=https://sonarcloud.io
 sonar.organization=rgposadas
 sonar.sources=libs, apps
-sonar.exclusions=**/*jest*.js,**/*spec*
+sonar.exclusions=**/*jest*.js,**/*spec*,**/mull-ui-e2e/**,**/database/**,**/environments/**,**/strategies/**, **/generated/**
```

Comment:
Idk if `strategies` should be excluded.

Reply:
The reason why I excluded `strategies` was because those were the Facebook, Twitter and Google redirects for login, so I don't think it's in our scope to cover those part with tests.